### PR TITLE
SQL script to copy anonymized data

### PR DESCRIPTION
**Pull request by sartan**
The included script
1. Creates an `anon` schema to house the anonymized data separately from "production"
   data, which lives in the `public` schema
1. Copies of certain tables (including data) from the `public` schema to the `anon` schema
1. Replaces personally identifiable information (PII) with anonymized values

Rest of the documentation can be found in [the readme](https://github.com/codeforamerica/vita-min/blob/b4b8a79f603597483fecf73c214bd2c0f6177d04/db/copy_anonymized_data.md).

### CODE REVIEW

**Comment by sniemeyer13** [comment]
Who is going to run this script? Product with the help of eng?

**Comment by sartan** [comment]
> Who is going to run this script? Product with the help of eng?

That was my thought, but curious to hear from others.

**Review by alaroia-cfa — APPROVED** [review]
This is great as-is. If we determine we need to remove ZIP code, too, we can do that in a follow-up.

Thanks for working on it!

**Comment by alaroia-cfa** [comment]
@sniemeyer13 @sartan I think that since the product doesn't have Aptible access, eng will have to run it. That's OK; https://www.pivotaltracker.com/story/show/175297684 is a chore not a story, so no need for acceptance.

File: db/copy_anonymized_data.sql [hunk 1]
```diff
@@ -0,0 +1,62 @@
+create schema if not exists anon;
+
+-- intake_site_drop_offs
+drop table if exists anon.intake_site_drop_offs;
+create table anon.intake_site_drop_offs as (select * from intake_site_drop_offs);
+update anon.intake_site_drop_offs set additional_info = 'ANONYMIZED';
```
[the remark this answers is not in the document]
Comment (sartan) [reply db/copy_anonymized_data.sql:6]: Thanks. The goal was to distinguish cases where data was anonymized from those where the data wasn't provided. I suppose we don't need to yell it though.

File: db/copy_anonymized_data.sql [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+update anon.dependents set birth_date = to_date(concat('1/1/', extract(year from birth_date)), 'MM/DD/YYYY');
+update anon.dependents set first_name = 'ANONYMIZED';
+update anon.dependents set last_name = 'ANONYMIZED';
+
+-- intakes
+drop table if exists anon.intakes;
+create table anon.intakes as (select * from intakes);
+update anon.intakes set additional_info = 'ANONYMIZED';
+update anon.intakes set email_address = 'ANONYMIZED';
+update anon.intakes set encrypted_bank_account_number = 'ANONYMIZED';
```
[the remark this answers is not in the document]
Comment (sartan) [reply db/copy_anonymized_data.sql:30]: Good eye!

File: db/copy_anonymized_data.sql [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+update anon.intakes set primary_consented_to_service_ip = '127.0.0.1';
+update anon.intakes set primary_first_name = 'ANONYMIZED';
+update anon.intakes set primary_last_name = 'ANONYMIZED';
+update anon.intakes set sms_phone_number = 'ANONYMIZED';
+update anon.intakes set spouse_birth_date = to_date(concat('1/1/', extract(year from spouse_birth_date)), 'MM/DD/YYYY');
+update anon.intakes set spouse_consented_to_service_ip = '127.0.0.1';
+update anon.intakes set spouse_email_address = 'ANONYMIZED';
+update anon.intakes set spouse_first_name = 'ANONYMIZED';
+update anon.intakes set spouse_last_name = 'ANONYMIZED';
+update anon.intakes set street_address = 'ANONYMIZED';
```
[the remark this answers is not in the document]
Comment (sartan) [reply db/copy_anonymized_data.sql:57]: Good call. Better safe than sorry, I'm going to anonymize for now and verify with product via tracker whether we should **not** be doing that.

**Pull request merged**